sos/policies/__init__.py

```diff
@@ -423,7 +433,6 @@ class Policy(object):
 should be reviewed by the originating organization before being passed to \
 any third party.
 
-No changes will be made to system configuration.
```

Comment:
I still think this needs to be dynamically reported based on the presence or not of the `--allow-system-changes` option.

By default, we don't make any changes. This is a long-time expectation from sosreport end-users, and just removing the line will cause concern among users in certain environments (government, banking, etc...).

Similarly, if `--allow-system-changes` is set, we should highlight that with something like `The --allow-system-changes option has been enabled, and so minor system changes such as kernel modules being loaded may take place`.

Reply:
Are we ok handling that in sosreport.py and not policies/__init__..

Reply:
I think it'd be fine in `policies/__init__.get_msg()`, and nothing is currently overriding it.

```python
    def get_msg(self):
        """This method is used to prepare the preamble text to display to
        the user in non-batch mode. If your policy sets self.distro that
        text will be substituted accordingly. You can also override this
        method to do something more complicated."""
        width = 72
        _msg = self.msg % {'distro': self.distro, 'vendor': self.vendor,
                           'vendor_url': self.vendor_url,
                           'vendor_text': self.vendor_text}

        if self.commons['cmdlineopts'].allow_system_changes:
            _msg += <some warning>
        else:
             _msg += <no changes>
```

Reply:
Thanks that helped a bunch!